cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,158 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-13977` - Removed duplicate classes to reduce the size of the sandbox by a couple hundred megabytes.
+
+- :cask-issue:`CDAP-14461` - Added cdap-env.sh to allow configuring jvm options for sandbox.
+
+- :cask-issue:`CDAP-13653` - Added support for Apache Spark 2.3.
+
+- :cask-issue:`CDAP-14013` - Added capability for external dataset to record their schema.
+
+- :cask-issue:`CDAP-14022` - Added a framework that can be used to bootstrap a CDAP instance.
+
+- :cask-issue:`CDAP-14091` - The Dataproc provisioner will try to pick up the project id and credentials from the environment if they are not specified.
+
+- :cask-issue:`CDAP-14104` - The Dataproc provisioner will use internal IP addresses when CDAP is in the same network as the Dataproc cluster.
+
+- :cask-issue:`CDAP-14168` - Added capability to always display current dataset schema in Field Level Lineage.
+
+- :cask-issue:`CDAP-13746` - Added the ability to configure system wide provisioner properties that can be set by admins but not by users.
+
+- :cask-issue:`CDAP-13886` - Improved error handling in Preparation.
+
+- :cask-issue:`CDAP-13924` - Added capability to allow specifying requirements by plugins and filter them on the basis of their requirements.
+
+- :cask-issue:`CDAP-13975` - Added REST endpoints to query the run counts of a program.
+
+- :cask-issue:`CDAP-14023` - Added a FileSink batch sink, FileMove action, and FileDelete action to replace their HDFS counterparts.
+
+- :cask-issue:`CDAP-14097` - Added a configurable jvm option to kill CDAP process immediately on sandbox when an OutOfMemory error occurs.
+
+- :cask-issue:`CDAP-14135` - Added better trace logging for dataset service.
+
+- :cask-issue:`CDAP-14260` - Added a REST endpoint to get the latest run record of multiple programs in a single call.
+
+- :cask-issue:`CDAP-14386` - Make Google Cloud Storage, Google Cloud BigQuery, and Google Cloud Spanner connection properties optional (project id, service account keyfile path, temporary GCS bucket).
+
+- :cask-issue:`CDAP-14401` - Google Cloud PubSub sink will try to create the topic if it does not exist while preparing for the run.
+
+- :cask-issue:`CDAP-14475` - Added csv, tsv, delimited, json, and blob as formats to the S3 source and sink.
+
+- :cask-issue:`CDAP-9048` - Added a button on external sources and sinks to jump to the dataset detail page.
+
+- :cask-issue:`CDAP-14040` - Added format and suppress query params to the program logs endpoint to match the program run logs endpoint.
+
+- :cask-issue:`CDAP-14132` - Made all CDAP examples to be compatible with Spark 2.
+
+- :cask-issue:`CDAP-14220` - Added worker and master disk size properties to the Dataproc provisioner.
+
+- :cask-issue:`CDAP-14298` - Improves operational behavior of the dataset service.
```

Comment:
Improved...

Although I wonder if we should remove it, unless we add more details